### PR TITLE
Introduce timestamp utilities

### PR DESCRIPTION
**Pull request by xbrianh**
depends on #203

### CODE REVIEW

File: tests/test_utils.py [hunk 1]
```diff
@@ -24,6 +25,11 @@ def my_func():
                 my_func()
             self.assertEqual(count['count'], expected_attempts)
 
+    def test_timestamps(self):
+        dt = datetime.utcnow()
+        ts = timestamp(dt)
+        self.assertEqual(datetime_from_timestamp(ts), dt)
+        timestamp_now()
```
Comment (DailyDreaming) [inline tests/test_utils.py:32]: Should this line have an assertion statement?

**Review by DailyDreaming — APPROVED** [review]
LGTM.  See comment.

File: tests/test_utils.py (hunk 1 above)
Comment (xbrianh) [reply tests/test_utils.py:32]: Yeah... I wasn't sure how to test this so I figured calling it is better than nothing.

Comment (xbrianh) [reply tests/test_utils.py]: assertion added!

**Pull request merged**